source/dub/dub.d

```diff
@@ -277,28 +277,104 @@ class Dub {
 
 		The script above can be invoked with "dub --single test.d".
 	*/
+		struct Parser {
```

Comment:
Oh sorry, I see now that this also checks the comment contents. But I guess that we could just separate the two using an additional `.filter!(...)` call and keep this struct to do only the generic comment parsing.